worker-api/src/main/java/com/hpe/caf/api/worker/TaskMessage.java

```diff
@@ -233,4 +234,34 @@ public void setPriority(Integer priority)
     {
         this.priority = priority;
     }
+
+    @Override
+    public String toString()
+    {
+        final StringBuilder builder = new StringBuilder();
+        builder.append("TaskMessage [version=");
+        builder.append(version);
+        builder.append(", taskId=");
+        builder.append(taskId);
+        builder.append(", taskClassifier=");
+        builder.append(taskClassifier);
+        builder.append(", taskApiVersion=");
+        builder.append(taskApiVersion);
+        builder.append(", taskData=");
+        builder.append(Arrays.toString(taskData));
```

Comment:
taskData is a byte array, I think Arrays.toString will corrupt the taskData.

Reply:
I saw you updated the test to get the bytes for a string, was that trying to show there was no problem?